serverless/wasm/main.go

```diff
@@ -0,0 +1,225 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build wasm
+
+// Package main provides a series of entrypoints for using a serverless log from
+// JavaScript in a browser.
+//
+// Build with:
+//   GOOS=js GOARCH=wasm go build -o serverless/wasm/main.wasm -tags wasm ./serverless/wasm/
+//
+// To view locally:
+//   # Copy Go wasm JS shim:
+//   cp "$(go env GOROOT)/misc/wasm/wasm_exec.js" serverless/wasm/
+//   # Run a webserver serving the wasm dir
+//   # e.g. using goexec (to install goexec: go get -u github.com/shurcooL/goexec):
+//   goexec 'http.ListenAndServe(`:8080`, http.FileServer(http.Dir(`serverless/wasm`)))'
+// Then point your browser at the server.
```

Comment:
Consider removing from here and replacing with a pointer to the README. This is asking to go out of sync.

Reply:
good call, done